source/libs/page-detect.js

```diff
@@ -35,6 +35,8 @@ export const getOwnerAndRepo = () => {
 	return {ownerName, repoName};
 };
 
+export const getPRNumber = () => isPR() && getRepoPath().replace(/^.*?(\d+).*$/, '$1');
```

Comment:
Can you add a test for this?

https://github.com/sindresorhus/refined-github/blob/master/test/page-detect.js

Reply:
Also it can be made generic:

```js
 export const getDiscussionNumber = () => (isPR() || isIssue()) && getRepoPath().split('/')[1];
```